app/services/spoonacular_service.rb

```diff
@@ -29,7 +29,7 @@ def complex_search_params(search_params)
       addRecipeInformation: true, fillIngredients: true, ignorePantry: true,
       minCalories: 0, minCholesterol: 0, minFat: 0, minProtein: 0, minCarbs: 0,
       minSugar: 0, minSodium: 0,
-      number: 10, sort: 'random'
+      number: 12, sort: 'min-missing-ingredients'
```

Comment:
I wish random worked the way that we intended.. I believe if you search for say turkey, honey, gravy - you'd always get the same 10 recipes - as opposed to there being 20 possible recipes and it selecting randomly from the list.